.travis.yml

```diff
@@ -28,6 +28,7 @@ before_install:
       llvm-${LLVM_VERSION}-dev
       llvm-${LLVM_VERSION}-tools
       libclang-${LLVM_VERSION}-dev
```

Comment:
This can probably be removed now

Reply:
We still need it because it contains headers, which we include:
https://github.com/intel/opencl-clang/blob/7258431b713090f1b10684befe4f675c4763b6c5/common_clang.cpp#L44-L52